packages/flipper-plugin-android-performance-profiler/src/__tests__/Plugin.test.ts

```diff
@@ -10,6 +10,11 @@ import * as Plugin from "..";
 global.electronRequire = require;
 require("@testing-library/react");
 
+// This is a dependency of flipper-common, but we hit SyntaxError: Unexpected token 'export' with jest >= 28 and uuid v < 9
+jest.mock("uuid", () => ({
+  v4: () => "uuid",
+}));
```

Comment:
I suppose we can put it in \_\_mocks\_\_

Reply:
I'd rather leave it here explicitely because it's only needed because of Flipper